src/components/layout/Duo/__specs__/Duo.spec.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+import Duo from '../Duo'
+
+describe('src | components | layout | Duo', () => {
+  it('should match snapshot with no render when isDisabled is true', () => {
```

Comment:
Ton intention est trop technique, il faut rester métier.
Je te propose :
```suggestion
  it('when this offer is a "offre duo" should render nothing', () => {
```